examples/federated-kaplan-meier-fitter/pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "1.0.0"
 description = "Federated Kaplan Meier Fitter with Flower"
 license = "Apache-2.0"
 dependencies = [
-    "flwr[simulation]>=1.10.0",
```

Comment:
Quick question: do we need to use `>=1.11.1` here since we are about to release `1.11.1` soon.

Reply:
@panh99 you're right, that'd be event better.